pkg/cosign/fulcio/fulcio.go

```diff
@@ -198,7 +200,16 @@ var Roots *x509.CertPool
 
 func init() {
 	cp := x509.NewCertPool()
-	if !cp.AppendCertsFromPEM([]byte(rootPem)) {
+	rootEnv := os.Getenv(altRoot)
+	if rootEnv != "" {
+		raw, err := ioutil.ReadFile(rootEnv)
```

Comment:
Oh sorry - I just realized this is a file, not a directory. I think we have two options:

- Keep it as a directory and concatenate all the pem files in it using something like filepath.Glob (this is pretty standard for roots, you can easily have multiple)
- Rename it to "_FILE" and leave it this way. 

My hunch in the long term is that we'll want the ability to have multiple (adding yours in addition to the public good one)